packages/react-core/src/components/Menu/MenuInput.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Menu/menu';
+
+export interface MenuInputProps extends React.HTMLProps<HTMLElement> {
+  /** Items within input */
+  children?: React.ReactNode;
+  /** Forwarded ref */
+  innerRef?: React.Ref<any>;
+}
+
+const MenuInputBase: React.FunctionComponent<MenuInputProps> = ({ children }: MenuInputProps) => (
+  <div className={styles.menuSearch}>{children}</div>
```

Comment:
I think I also saw Core nesting `pf-c-menu__search-input` here, is that needed @mcoker ?

Reply:
@jschuler it's not necessary, but having that element makes it easier for us to style the text input relative to other things we might add to the menu search element.